datadog_checks_base/datadog_checks/base/ddyaml.py

```diff
@@ -40,14 +38,15 @@ def safe_yaml_dump_all(
     tags=None,
 ):
     if Dumper != yDumper:
-        stream_name = get_stream_name(stream)
-        log.debug("Unsafe dumping of YAML has been disabled - using safe dumper instead in %s", stream_name)
+        log.debug(
+            "`%s` YAML dumper is used instead of the default one, please make sure it is safe to do so", Dumper.__name__
+        )
```

Comment:
Would it make sense to allow either SafeDumper or CSafeDumper without a debug log, since that seems to be the motivating case here?

Reply:
I think it's better to have the debug line whenever the loader is different than the default one (even when using SafeDumper instead of CSafeDumper). It shouldn't happen often, so we should notify it in the logs. If the log line becomes too noisy we can still change it in the future.